src/Utils/ItemAlignment.vala

```diff
@@ -144,6 +148,15 @@ public class Akira.Utils.ItemAlignment : Object {
                     }
                 }
 
+                break;
+            case AlignmentType.ANCHOR:
+                unowned var bb = anchor.instance.bounding_box;
+
+                align_to.left = bb.left;
+                align_to.right = bb.right;
+                align_to.top = bb.top;
+                align_to.bottom = bb.bottom;
+                found_bound = true;
```

Comment:
Little English nit: every time we use the word `bound` it should actually be `bounds` as it represents the boundaries of an object, so it should be plural.